arui-demo/components/preview-component.jsx

```diff
@@ -0,0 +1,29 @@
+import { Component, PropTypes as Type } from 'react';
```

Comment:
@GREENpoint, https://github.com/styleguidist/react-styleguidist/blob/master/src/rsg-components/Preview/Preview.js
this file contains 2 components (one private and one public). We need override both components